modules/kubernetes-addons/aws-privateca-issuer/locals.tf

```diff
@@ -32,8 +32,8 @@ locals {
   ]
 
   irsa_config = {
-    create_kubernetes_namespace       = true
-    kubernetes_namespace              = local.name
+    create_kubernetes_namespace       = try(local.helm_config["create_namespace"], true)
+    kubernetes_namespace              = local.helm_config["namespace"]
```

Comment:
we should hoist this up to a local variable and do the same as https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/595/files#r905993662 to ensure all references are aligned (including within helm config)